environments/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:22.04
+FROM ubuntu:latest
```

Comment:
I think we should pin a version, in case we get unexpected behaviour in future.

Reply:
We could, but the only unexpected behaviour ever seen here was around python versions being bumped. This has essentially 0 dependencies, so should be fine

Reply:
Fair enough. I felt I had to check. 😸